config/_default/menus/menus.en.yaml

```
@@ -3379,6 +3379,11 @@ main:
     parent: security_platform
     identifier: security_notifications
     weight: 6
+  - name: Threat Intel
```

```suggestion
  - name: Threat Intelligence
```

For consistency, as we use the full name elsewhere